src/Client.ts

```diff
@@ -111,6 +115,174 @@ export class Client {
             });
     }
 
+    public getCachedStatus(vin: string, callback: (err?: Error, status?) => void): void {
```

Comment:
Would you mind adding brief descriptions of these new functions in the API list in the `README.md`?

Reply:
Yes sure. It might not be until later on today but I'll get it updated.